server/models/video/video-format-utils.ts

```diff
@@ -220,6 +223,17 @@ function addVideoFilesInAPAcc (
       fps: file.fps
     })
 
+    if (file.metadata) {
```

Comment:
So we don't have to fetch metadata every time we want to create an AP JSON object, I think we should always include it in the object.

Reply:
What do you mean?